src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj

```diff
@@ -6,15 +6,14 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <Version>3.2.0</Version>
+    <Version>3.3.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;REST HTTP client;Computer Vision;Computer Vision API;Computer Vision SDK;Vision;netcore451511</PackageTags>
     <PackageReleaseNotes>2.0.0 - A breaking change for RecognizeText.  Rather than a boolean argument for handwritten text, callers must specify a mode enumerated value.</PackageReleaseNotes>
     <PackageReleaseNotes>
```

Comment:
can you consolidate relese notes. it should have only 1 tag

Reply:
@shahabhijeet I addressed your feedback.